vms/proposervm/vm_test.go

```diff
@@ -1600,11 +1589,7 @@ func TestRejectedHeightNotIndexed(t *testing.T) {
 	proVM := New(
 		coreVM,
 		Config{
-			Upgrades: upgrade.Config{
-				ApricotPhase4Time:            time.Unix(0, 0),
-				ApricotPhase4MinPChainHeight: 0,
-				DurangoTime:                  time.Unix(0, 0),
-			},
+			Upgrades:            upgradetest.GetConfigWithUpgradeTime(upgradetest.Latest, time.Unix(0, 0)),
```

Comment:
I know it's no the same, as one is 1970 and the other is just 0, but, why the discrepancy between time.Unix(0,0) and time.Time? Won't this test work with time.Time{} ?